dmutils/presenters.py

```diff
@@ -28,3 +28,10 @@ def _upload(self, value):
             "url": value or "",
             "filename": value.split("/")[-1] or ""
         }
+
+    def _boolean(self, value):
+        if value is True:
```

Comment:
It would, yes, if any old stuff gets passed in to it.  In that case maybe it should be `if value == True` - the `is` operator is for testing object equality, not value equality. (It will almost certainly work here, but is not guaranteed to in future if implementation of Python changes).